bin/symfony_requirements

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env php
+<?php
```

Comment:
I do think it is useful, to check the PHP CLI configuration from inside the Vagrant box.